src/pudl_zenodo_storage/frictionless/epacems_unitid_eia_plant_crosswalk.py

```diff
@@ -0,0 +1,25 @@
+# -*- coding: utf-8 -*-
```

Comment:
We're only supporting Python 3, which uses UTF encoding by default, so this first comment line can/should be removed wherever you see it, and not added to new files.